DataSetExplorer/DataSetBuilder/Model/Repository/IDataSetAnnotationRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DataSetExplorer.DataSetBuilder.Model.Repository
+{
+    public interface IDataSetAnnotationRepository
+    {
+        DataSetAnnotation GetDataSetAnnotation(int id);
+        Annotator GetAnnotator(int id);
+        CodeSmell GetGodeSmell(string value);
```

Comment:
Typo: GodeSmell
What is this value? Is it code smell name (like "Long method")? Maybe it should be renamed.